tests/psycopg2_test.py

```diff
@@ -41,7 +42,8 @@ class SpecifySocketTestCase(unittest.TestCase):
 
     def setUp(self):
         self.socket_dir = tempfile.mkdtemp()
-        self.db = TempDB(databases=['test_db'], verbosity=1,
+        os.chmod(self.socket_dir, 0o777)
```

Comment:
(sorry if I asked this before) -- now that we're using `--user=postgres` does the `chmod` still need to be `777` or could it drop down to something more restrictive?  Or is this needed so for the inner dockerized postgres so that it can write into the temp directory (which will have a random uid/gid assigned to it from the perspective of the containerized pg)?   I'm guessing it's the latter.  If that's correct then it might be worth a comment since the details can be easy to forget down the line.

Reply:
yes it is the latter, i'll add a comment